contracts/contracts/vault/VaultAdmin.sol

```diff
@@ -250,6 +280,77 @@ contract VaultAdmin is VaultStorage {
         }
     }
 
+    /**
+     * @notice Strategist swaps collateral assets sitting in the vault.
+     * @param _fromAsset The token address of the asset being sold by the vault.
+     * @param _toAsset The token address of the asset being purchased by the vault.
+     * @param _fromAssetAmount The amount of assets being sold by the vault.
+     * @param _minToAssetAmount The minimum amount of assets to be purchased.
+     * @param _data implementation specific data. eg 1Inch swap data
+     * @return toAssetAmount The amount of toAssets that was received from the swap
+     */
+    function swapCollateral(
+        address _fromAsset,
+        address _toAsset,
+        uint256 _fromAssetAmount,
+        uint256 _minToAssetAmount,
+        bytes calldata _data
+    )
+        external
+        whenNotCapitalPaused
```

Comment:
I realized I did not communicate clearly in the design decisions on this one. We want this method to match the other _strategist_ allocation methods like `depositTo` and `withdrawFrom` which can operate even when the capital is paused.

We should remove `whenNotCapitalPaused` from this one.

Reply:
Yes, I think that makes sense. The strategist can swap out a depegging asset. I'll do the change